src/libraries/Microsoft.PowerFx.Core/Texl/Builtins/Table.cs

```diff
@@ -74,7 +74,7 @@ public override bool CheckTypes(CheckTypesContext context, TexlNode[] args, DTyp
                 }
                 else
                 {
-                    if (DType.TryUnionWithCoerce(rowType, argType, context.Features.PowerFxV1CompatibilityRules, out var newType, out bool coercionNeeded))
+                    if (DType.TryUnionWithCoerce(rowType, argType, context.Features.PowerFxV1CompatibilityRules, coerceToLeftTypeOnly: context.Features.StronglyTypedBuiltinEnums, out var newType, out bool coercionNeeded))
```

Comment:
>StronglyTypedBuiltinEnums [](http://example.com/codeflow?start=152&length=25)

Should this be PowerFxV1CompatibilityRules?

Reply:
Not really - this is purely a strongly-typed enum issue. If we have PFx rules disabled (like in Power Apps), but strongly-typed enums enabled, then we can allow [ErrorKind.Div0, 13] to work (since they're both numbers).